Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -236,6 +244,7 @@
   - type: Clothing
     sprite: Objects/Weapons/Guns/Battery/laser_gun.rsi
   - type: Gun
+    fireRate: 2 # DeltaV - parent changed to 1.5
```

Minor nitpick, but I think the explanation on the comment is confusing, I'd probably remove it from here and the other places

```suggestion
    fireRate: 2 # DeltaV
```